index.bs

```diff
@@ -99,7 +99,7 @@ interface FaceDetector {
   <dt><dfn constructor for="FaceDetector"><code>FaceDetector(optional FaceDetectorOptions faceDetectorOptions)</code></dfn></dt>
   <dd>Constructs a new {{FaceDetector}} with the optional |faceDetectorOptions|.</dd>
```

Comment:
Please add to the index.bs `metadata` an entry `Translation` (search for `Translation` in https://github.com/mikewest/bikeshed/blob/master/docs/metadata.md).

Actually, WDYT if we 
a) land the modifications to `index.bs`  (and `index.html`) now, in this PR
b) move the addition of the chinese translation to _another_ PR, which will include adding the `Translation` entry in `index.bs` (and there should be another correspondent `Translation` entry in index-zh-cn.bs` indicating the english version).
That way it'll be easier to track the history of changes.